### PR TITLE
feat: attach/detach, loop animation, stop animation/sound

### PR DESCRIPTION
**Pull request by cazala**
This PR adds support for the `attach_to_player` and `detach_from_player` actions, adds support for the `loop` property in `play_animation`, and adds support for the new `stop_animation` and `stop_sound` actions.

### CODE REVIEW

File: packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    label: 'Play Once',
+    value: PLAY_MODE.PLAY_ONCE
+  },
+  {
+    label: 'Loop',
+    value: PLAY_MODE.LOOP
+  }
+]
+
+const PlaySoundAction: React.FC<Props> = ({ value, animations, onUpdate }: Props) => {
```
Comment (cyaiox) [inline packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx:27]:
```suggestion
const PlayAnimationAction: React.FC<Props> = ({ value, animations, onUpdate }: Props) => {
```

File: packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  const handleChangePlayMode = useCallback(
+    ({ target: { value } }: React.ChangeEvent<HTMLSelectElement>) => {
+      setPayload({ ...payload, loop: value === PLAY_MODE.LOOP })
+    },
+    [payload, setPayload]
+  )
+
+  return (
+    <div className="PlaySoundActionContainer">
```
Comment (cyaiox) [inline packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx:52]:
```suggestion
    <div className="PlayAnimationActionContainer">
```

File: packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.PlayanimationActionContainer {
```
Comment (cyaiox) [inline packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.css:1]:
```suggestion
.PlayAnimationActionContainer {
```

File: packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx (hunk 2 above)
Comment (cazala) [reply packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx]: done 👍

File: packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx (hunk 1 above)
Comment (cazala) [reply packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/PlayAnimationAction/PlayAnimationAction.tsx:27]: done 👍

**Review by cyaiox — APPROVED** [review]

**Pull request merged**